devel/libiodbc/Portfile

```diff
@@ -4,37 +4,39 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           active_variants 1.1
 
-github.setup        openlink iODBC 3.52.15 v
+github.setup        openlink iODBC 3.52.16 v
+github.tarball_from releases
 #override name (keep it lowercase)
 name                libiodbc
-revision            2
+revision            0
 categories          devel
 maintainers         {snc @nerdling} openmaintainer
 license             BSD
 description         Independent Open DataBase Connectivity
 long_description    iODBC is the acronym for Independent Open DataBase Connectivity, \
                     an Open Source platform independent implementation of both the \
                     ODBC and X/Open specifications.
-platforms           darwin
 
-checksums           rmd160  60a54f0f032e98c3dd56190044202e7d942a9eeb \
-                    sha256  40c96973dfa02725f3e338eeed2f735e662eeb59de43a58ec871a34dc64c698d \
-                    size    659032
+distname            ${name}-${version}
```

Comment:
Believe this is also a default value.

Reply:
GitHub port group overrides it with the project name.